src/content-blocks/contact.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Contact
+permalink: false
+navUrl: contact
+subtitle: 
+layout: page.njk
+order: 6
+
+author:
+  image: 
+  name: Overte e.V.
+  date: 05.02.2022
+tags: contentBlock
+articleTopLinks: {
+  
+}
+---
+
+
+Join us on our [Matrix](https://matrix.to/#/#overte:matrix.org) and [Discord](https://discord.gg/4YuQvc8K2f)
```

Comment:
I don't think we should advertise Discord.
It is proprietary, insecure, and will break compatibility again eventually. For all we know, they might even disallow bridging in the future.

Reply:
I have no idea how to solve this. Most of my friends don't and won't have Matrix. I'm tired of begging again and again for this.

Reply:
I don't want to exclude people from already small community.

Reply:
I'm giving up on this for now.

Reply:
I also removed the build directory